### PR TITLE
Large Health Doll

### PR DESCRIPTION
**Pull request by Iamgoofball**
basically a health doll but bigger and your combat selector was merged into it
ss 2015-12-06 at 05 19 41
ss 2015-12-06 at 05 21 22

@WJohn did all the sprites fam

### CODE REVIEW

**Comment by WJohn** [comment]
I'm still not very happy with the sprites I made, but I can edit them in the future to look better.

**Comment by tkdrg** [comment]
it merges the health doll and the targeting thing into one bigger clicky colory thingy
its good

**Comment by Anonus** [comment]
The current sprite for this, with a little editing, could probably be used as a spooky alien of some sort, wj.

File: code/_onclick/hud/hud.dm [hunk 1]
```diff
@@ -238,7 +239,7 @@ var/datum/global_hud/global_hud = new()
 			action_intent.screen_loc = ui_acti //Restore intent selection to the original position
 			mymob.client.screen += mymob.zone_sel				//This one is a special snowflake
 			mymob.client.screen += mymob.healths				//As are the rest of these.
-			mymob.client.screen += mymob.healthdoll
+//			mymob.client.screen += mymob.healthdoll
```
Comment (optimumtact) [inline code/_onclick/hud/hud.dm]: remove

File: code/_onclick/hud/human.dm [hunk 2]
```diff
@@ -261,9 +261,9 @@
 	mymob.healths.name = "health"
 	mymob.healths.screen_loc = ui_health
 
-	mymob.healthdoll = new /obj/screen()
+/*	mymob.healthdoll = new /obj/screen()
```
Comment (optimumtact) [inline code/_onclick/hud/human.dm]: REMOVE

Comment (Iamgoofball) [inline code/_onclick/hud/human.dm]: never

this is MY code

**Comment by PKPenguin321** [comment]
its creepin me out

**Comment by Zilenan91** [comment]
it has no feet, and coupled with the mouth this is something I'd see if I was having sleep paralysis.

**Comment by Incoming5643** [comment]
legs seem short for the torso

**Comment by Anonus** [comment]
It is very creepy and should be used as an alien mob thing or maybe a huge statue

**Comment by JJRcop** [comment]
Shouldn't have thumbs, they look weird.

**Comment by Zilenan91** [comment]
Ok yeah holy fuck looking back at this this is some seriously uncanny valley shit. Something you'd see in Lifeweb aesthetic rather than SS13 aesthetic.

**Comment by WJohn** [comment]
I mean I did do the entire thing in 3 hours, the actual pose was maybe one hour so I could definitely improve it significantly.

**Comment by Incoming5643** [comment]
According to five minutes of google searching apparently the leg to torso ratio should be somewhere around 45%, the mode up there was at like 38%. Here's a hack job of what ~45% legs look like.

d10_icon

**Comment by Nienhaus** [comment]
That seems to long.

**Comment by Incoming5643** [comment]
I don't think there's a way for this not to look wonky because the figure is about six heads tall, which is JUUUUUUST outside the realms of realism. At the same time the head kinda needs to be big for the body because there's all sorts of tiny little selection zones there that are almost depreciated but still used in a few places.

**Comment by Shapsy** [comment]
What really creeps me out about it is how it looks like a cadaver

**Comment by Incoming5643** [comment]
Go full bayonetta
d10_icon

**Comment by Incoming5643** [comment]
I'm gonna guess that your perception of the legs is gonna depend on whether you think the dolls feat are either hanging slack down, or not drawn on the model. I was working off the notion that it was the former.

**Comment by PKPenguin321** [comment]
why not just use the health doll sprite we already have
its already bigger than the targeting doll so clickability isnt an issue

**Comment by Nienhaus** [comment]
untitled
I tried

**Comment by PKPenguin321** [comment]
but why do we even need this sprite just use the health doll we already have reee

**Comment by Zilenan91** [comment]
Okay the one with feet is so much better. I could deal with that one. They just look too alien and creepy without feet.

Only complaint is the mouth in the o position looks a bit odd but that's just a minor gripe that doesn't really need to be fixed.

**Comment by Anonus** [comment]
# USE THE CREEPY ONE AS AN ALIEN STATUE OR SOMETHING COME ON IT'S SUPER CREEPY IT'D BE PERFECT AS A STATUE

**Comment by Zilenan91** [comment]
Enlarged version without the border for use in away missions Y/Y?

**Comment by Anonus** [comment]
It doesn't need to be larger, it's already 32x64 and has about enough room for a statue base.

**Comment by Shadowlight213** [comment]
Yeah this sprite is right in the uncanny valley. Really really creepy. :-1: to it being a health doll replacement.

**Comment by neersighted** [comment]
:+1: @Nienhaus's version looks good to me.

**Comment by YotaXP** [comment]
Had to take a whack at it myself

**Comment by kingofkosmos** [comment]
Awesome, but make it look less realistic and more similar to the current doll, like a simple [pictogram man](https://www.google.fi/search?q=pictogram&espv=2&biw=784&bih=811&source=lnms&tbm=isch&sa=X&ved=0ahUKEwjxsuKUm8nJAhXGFiwKHTXICbwQ_AUIBigB#tbm=isch&q=pictogram+man).

**Comment by Shadowlight213** [comment]
Much much better :+1:

**Comment by optimumtact** [comment]
Where is pictowoman?

**Comment by Ausops** [comment]
>Where is pictowoman?

I kind of got you fam

https://imgur.com/c9P6Cm2
https://imgur.com/yIz7soS

I found some old sprites of mine that I touched up for this. I know they might not fit but I am past caring, so I don't mind if they're used or not. Iconstates are all done except I realized the damage for chest and groin were merged instead of separate but I kept them separate anyway. Male ones have M added to the end of their iconstate, female ones have F added.

**Comment by PKPenguin321** [comment]
why not just use the current health doll reeee

**Comment by optimumtact** [comment]
this one is bigger making it easier to click on, it's easily a major improvement

**Comment by Dorsisdwarf** [comment]
Yota's is probbably the best.

But only if you make the doll have a sad face when badly hurt

**Comment by Jacquerel** [comment]
YotaXP's one looks so happy, I love it

**Comment by Aranclanos** [comment]
I think it looks weird because the head needs to be 1/7 of the body on height if I recall correctly

**Comment by Steelpoint** [comment]
Whatever happens make sure there is no awkward 'gap' in the middle of the two knee's.

**Comment by Dorsisdwarf** [comment]
Yota's new plasmaman/lizard healthdolls are great

**Comment by Iamgoofball** [comment]
Paprika why'd you delete the branch

**Comment by Dorsisdwarf** [comment]
I think he deleted all of them

**Comment by optimumtact** [comment]
I'll restore it to my branch

**Comment by optimumtact** [comment]
https://github.com/optimumtact/-tg-station/tree/largedoll

**Comment by tkdrg** [comment]
@Iamgoofball @optimumtact @paprka someone fix this

**Comment by Incoming5643** [comment]
it's still kind of an open question which doll to use

**Comment by optimumtact** [comment]
Yota's are the best

**Comment by optimumtact** [comment]
I'll take a look at this sometime this weekend

**Comment by tkdrg** [comment]
@optimumtact did you have the time to work on this?

**Comment by optimumtact** [comment]
no

**Comment by vista-** [comment]
:+1: for Yota's sprites

**Comment by tkdrg** [comment]
someone code this

**Pull request closed without merging**